source_handlers_test.go

```diff
@@ -1984,6 +1984,34 @@ func TestUnpauseSourceAndItsApplicationsInvalidTenant(t *testing.T) {
 	templates.NotFoundTest(t, rec)
 }
 
+// TestUnpauseSourceAndItsApplicationsTenantNotExists tests that not found is returned
+// for not existing tenant
+func TestUnpauseSourceAndItsApplicationsTenantNotExists(t *testing.T) {
+	testutils.SkipIfNotRunningIntegrationTests(t)
+	tenantId := fixtures.NotExistingTenantId
+	sourceId := int64(1)
```

Comment:
I know it is on more places but should not we take it from fixtures when we have we defined ?

Reply:
Ok it sounds good

Reply:
(anyway I would prefer to stop using constants - maybe we can discuss it later as it is wider thing)